app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -561,6 +563,13 @@ function AcuantCapture(
       setAcuantFailureCookie(null);
 
       onCameraAccessDeclined();
+
+      // Due to a bug with Safari on iOS we force the page to refresh on the third
+      // time a user denies permissions.
+      onFailedCameraPermissionAttempt();
+      if (failedCameraPermissionAttempts > 2) {
+        window.location.reload();
```

Comment:
Not sure if it's possible to reach this point with a "dirty" page state that would otherwise trigger the "are you sure you want to navigate?" prompt, but maybe we should [`removeUnloadProtection`](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/url/remove-unload-protection.ts) or [`forceRedirect`](https://github.com/18F/identity-idp/blob/main/app/javascript/packages/url/force-redirect.ts) just in case?

Reply:
Those are interesting, thanks for the links. In this situation, there's no way for a confirmation prompt, but I added ([commit](https://github.com/18F/identity-idp/pull/9260/commits/ff4d1293a3785df441e3c48f51427eb51dfe5fe4)) `removeUnloadProtection` just in case.